### PR TITLE
added api functionality for members

### PR DESCRIPTION
**Pull request by harshsoni-harsh**
Issue no. #41 
Fixed issues for cards.

### CODE REVIEW

**Comment by SlightlyEpic** [comment]
You forgot to remove console.logs

File: src/scripts/components/members.js [hunk 1]
```diff
@@ -1 +1,160 @@
...
+            loadMembers(event.target.value);
+        })
+    });
+}
+
+export default function membersjs() {
+    // members Section
+    loadMembers().then(()=>
+    {
+        main();
```
Comment (slightlyepic) [inline src/scripts/components/members.js]: This will call `loadMembers` twice

File: src/scripts/components/members.js [hunk 2]
```diff
@@ -1 +1,160 @@
...
+async function loadMembers(role) {
+    const membersDivCarousel = document.getElementById("membersDivCarousel");
+    membersDivCarousel.textContent = "";
+
+    const projectApi = document.createElement("div");
+    projectApi.classList = "team-members-container flex gap-16 w-[300px*6] animate-scroll";
+    membersDivCarousel.appendChild(projectApi);
+
+    try {
+        const response = await fetch("../../api/members.json");
```
Comment (slightlyepic) [inline src/scripts/components/members.js]: Also probably cache this to avoid resending requests to fetch the same data

**Comment by harshsoni-harsh** [comment]
I've made improvements in code regarding fetch calls, not it will be called only once at the start. Now you can merge it to the main repo.

**Review by Nancy-30 — APPROVED** [review]

**Pull request merged**